.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "parser": "@typescript-eslint/parser", // Specifies the ESLint parser
+    "extends": [
+        "plugin:@typescript-eslint/recommended", // Uses the recommended rules from the @typescript-eslint/eslint-plugin
```

Comment:
These are basic rules. You better activate something more strict like https://www.npmjs.com/package/eslint-config-airbnb

Reply:
Done.